cmd/kubeadm/app/cmd/phases/kubelet.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
+	kubeletphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubelet"
+	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
+	configutil "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
+	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	nodeutil "k8s.io/kubernetes/pkg/util/node"
+	"k8s.io/kubernetes/pkg/util/normalizer"
+)
+
+var (
+	kubeletWriteInitConfigLongDesc = normalizer.LongDesc(`
+		Write init kubelet configuration to disk for dynamic kubelet configuration feature, functionally equivalent to what implemented by kubeadm init.
```

Comment:
`kubeadm init` executes both `write-init-config` and `upload-dynamic-config`, so I will remove _"functionally equivalent to what implemented by kubeadm init"_ from the description of commands implementing only one of those actions.

As additional background on this comment, please note that in other phase command the sentency "functionally equivalent to what implemented by kubeadm init" is typically used only with the `all` subcommand that acts as a wrapper for all the available actions (e.g. `kubeadm phase kubeconfig all`).

Reply:
Totally reasonable. Agreed. Thanks!

Reply:
done.